### ISSUE TITLE
Make unread notifications more eye-catching

### ISSUE THREAD

**Issue by samk-acw** [netbox-community/netbox#17222]
### NetBox version

v4.1-beta1

### Feature type

Change to existing functionality

### Proposed functionality

The new notification functionality looks promising, but the only indicator I see is the small bell icon in the top-right corner, which is quite easy to miss if you're not consciencely looking for it.

Some options for improvement I could think of:

- Make the bell icon larger
- Change the colour for unread notifications - the teal colour blends in with the new colour scheme, red might stand out more
- Reposition the icon location - next to the search bar would be a more central area that is looked at more often than the top-right corner
- On page load, if there's unread notifications, automatically open the dropdown menu that shows the notification details
- More extreme option of the above - on page load, show the unread notification details in a modal/popup.

### Use case

Notifications should be obvious to the user, otherwise they can be missed.

### Database changes

none

### External dependencies

none

**Comment by jeremystretch** [comment]
I agree that the icon is a bit small. We could also look into an [animation](https://tabler.io/docs/components/icons#icon-animations) but that's probably overkill.

**Comment by abhi1693** [comment]
@jeremystretch I'd like to take this. I can implement point 1 and point 2 from the description. Everything else seems to be a broader scope for the implementation. If you agree, let me know.
